plugins/pulp_rpm/plugins/importers/iso/sync.py

```diff
@@ -229,7 +234,9 @@ def perform_sync(self):
                     iso.save()
                 except NotUniqueError:
                     iso = iso.__class__.objects.filter(**iso.unit_key).first()
-                self.add_catalog_entries([iso])
+                else:
+                    # Only if iso is from pulp manifest
```

Comment:
I don't understand this comment.  What does not getting an `NotUniqueError` have to do with a pulp manifest?